src/main/resources/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStep/help.html

```diff
@@ -0,0 +1,7 @@
+<pre><code>
```

Comment:
This would be the right place to document the fundamental limitation of the current implementation: that

```groovy
node {
  sshagent(…) {
    sh '''
sleep 999 # ← restart Jenkins here
ssh …
'''
  }
}
```

will not work.